lib/ansible/modules/cloud/ovirt/ovirt_storage_domains.py

```diff
@@ -494,6 +524,11 @@ def main():
         posixfs=dict(default=None, type='dict'),
         glusterfs=dict(default=None, type='dict'),
         fcp=dict(default=None, type='dict'),
+        discard_after_delete=dict(type='bool', default=False),
```

Comment:
Why not use default `None` and let backend default value?

Reply:
good point, will do